doc/source/whatsnew/v0.22.0.txt

```diff
@@ -94,6 +94,7 @@ Documentation Changes
 Bug Fixes
 ~~~~~~~~~
 
+- BUG: fillna maximum recursion depth exceeded in cmp (GH18159).
```

Comment:
Format the issue like

    :issue:`18159`

so that the link is rendered;.

Also, you can move this to 0.21.1 since it's a bugfix.

Reply:
Done.